lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.html

```diff
@@ -6,7 +6,7 @@
                              [attr.data-automation-id]="'adf-metadata-group-properties'">
             <mat-expansion-panel-header>
                 <mat-panel-title>
-                    <h1 class="adf-metadata-properties-title">{{ 'CORE.METADATA.BASIC.HEADER' | translate }}</h1>
+                    <h2 class="adf-metadata-properties-title">{{ 'CORE.METADATA.BASIC.HEADER' | translate }}</h2>
```

Comment:
please use the regular text here and move the class to the parent element, for example:

```
<mat-panel-title class="adf-metadata-properties-title">
{{ 'CORE.METADATA.BASIC.HEADER' | translate }}
</mat-panel-title>
```